comfy_cli/command/generate/app.py

```
@@ -0,0 +1,370 @@
+
+
+def _print_top_help() -> None:
+    """Custom help that emphasizes the model-first UX over Typer's auto-help."""
+    rprint("[bold]comfy generate[/bold] — call ComfyUI cloud models")
+    rprint("")
+    rprint("[bold]Usage:[/bold]")
+    rprint("  comfy generate <model> [--<param> value]... [--download PATH] [--async] [--api-key KEY]")
+    rprint("")
+    rprint("[bold]Examples:[/bold]")
+    rprint('  comfy generate flux-pro --prompt "a cat on the moon" --width 1024 --height 1024 --download cat.png')
+    rprint('  comfy generate ideogram-edit --image cat.png --mask m.png --prompt "add sunglasses" --rendering_speed TURBO')
```

Pylint (4.0.5)

[convention] Line too long (123/120)

(C0301)